sdk/tables/azure-data-tables/src/table_clients.cpp

```diff
@@ -311,7 +311,7 @@ Azure::Response<Models::Table> TableServiceClient::CreateTable(
     Core::Context const& context)
 {
   auto url = m_url;
-  url.AppendPath("Tables");
+  url.AppendPath(Azure::Core::Url::Encode("Tables"));
```

Comment:
nit: Constant alphanumeric ASCII strings like these don't need to be encoded, since they are unchanged (or already encoded).

Consider using a shared constant by defining this in one place and re-use.

Reply:
I know, but it's easier and clearer to just go full blast and pepper encodes everywhere, either we encode or not ,